engine-standalone-storage/src/relayer_db/mod.rs

```diff
@@ -69,9 +71,9 @@ pub fn initialize_transactions<I>(
 where
     I: FallibleIterator<Item = types::TransactionRow, Error = postgres::Error>,
 {
-    let signer_account_id = "relayer.aurora".parse().unwrap();
-    let predecessor_account_id: AccountId = "relayer.aurora".parse().unwrap();
-    let current_account_id = "aurora".parse().unwrap();
+    let signer_account_id = AccountId::from_str("relayer.aurora").expect("valid account_id");
+    let predecessor_account_id = AccountId::from_str("relayer.aurora").expect("valid account_id");
+    let current_account_id = AccountId::from_str("aurora").expect("valid account_id");
```

Comment:
It looks redundant since we create `AccountId` from static strings, which represent VALID account ids.

Reply:
Addressed in https://github.com/aurora-is-near/aurora-engine/pull/916/commits/28be3e2a84ba8797f1f8838f4e45f1e98e16a1f9